### ISSUE TITLE
`ObMasterLayoutConfig.focusableFragments` should handle `undefined` case w/o warning

### ISSUE THREAD

**Issue by rlmmc** [oblique-bit/oblique#147]
### Oblique version

12.0.1

### Browser & version

Firefox 132.0.1

### Node version

22.11.0

### Is this a regression?

Yes

### Description

```true
Whenever I go to a page without a hash at the end, i.e., `http://localhost:4200/overview`, I get a warning in the console stating the following:


undefined is not in the whitelist of ids of fragments that are allowed to be focused:
 content, navigation
 The whitelist of fragments that are allowed to be focused is defined in ObMasterLayoutConfig.focusableFragments oblique.mjs:10131:16
```
```


### Reproduction

```true
See above.
```

### Expected behaviour

```true
If there's no hash, don't try to focus nothing.
```

**Comment by nina-egger** [comment]
Hello @rlmmc 

That issue is known to us and a fix is planned version 12.0.2.
Note: Issue is tracked in ticket OUI-3373

**Comment by nina-egger** [comment]
This has been fixed with the version 12.0.2
